.github/workflows/build-publish.yaml

```diff
@@ -10,6 +10,10 @@ on:
         description: "Release tag"
         required: true
         default: ""
+      release_channel:
+        description: "Release channel"
+        required: true
+        defauilt: "alpha"
```

Comment:
typo here, sorry didn't catch this before

Reply:
I'll fix in my next PR